lib/scripts.js

```diff
@@ -82,12 +83,21 @@ exports.paths = {
  */
 exports.run = (script, callback) => {
   childProcess.execFile(script, (error, stdout, stderr) => {
+
+    // Platform scripts might output warnings to `stderr`
+    // that we don't want to take into consideration
+    // if the script exits with an exit code 0.
+    //
+    // If the scripts exits with a non-zero code, `.execFile()`
+    // yields an error whose message includes the `stderr`
+    // output, so we don't need to do anything ourselves.
+
     if (error) {
       return callback(error);
     }
 
     if (stderr.trim().length) {
-      return callback(new Error(stderr));
+      debug('stderr: %s', stderr);
```

Comment:
Sadly not, because its a standalone function. This would have been possible if the function returns an object containing a `.log()` function for instance.